File: image-selector-frontend/src/app/models/imagemDto.ts
export class ImagemDto {

  private imagemUrl: String;

  private tagsToRemove: String[];

  private tagsToAdd: String[];

  constructor(imagemUrl: String, tagsToRemove: String[], tagsToAdd: String[]) {
    this.imagemUrl = imagemUrl;
    this.tagsToRemove = tagsToRemove;
    this.tagsToAdd = tagsToAdd;
  }

  public getImagemUrl(): String {
    return this.imagemUrl;
  }

  public getTagsToRemove(): String[] {
    return this.tagsToRemove;
  }

  public getTagsToAdd(): String[] {
    return this.tagsToAdd;
  }

  public setImagemId(imagemUrl: string) {
    this.imagemUrl = imagemUrl;
  }

  public setTagsToRemove(tagsToRemove: String[]) {
    this.tagsToRemove = tagsToRemove;
  }

  public setTagsToAdd(tagsToAdd: String[]) {
    this.tagsToAdd = tagsToAdd;
  }
}

File: image-selector-backend/src/main/java/com/framed/imageselector/models/ImagemDto.java
package com.framed.imageselector.models;

public class ImagemDto {

    private String imagemUrl;

    private String[] tagsToRemove;

    private String[] tagsToAdd;

    public ImagemDto (String url, String[] tagsToRemove, String[] tagsToAdd) {
        this.imagemUrl = url;
        this.tagsToRemove = tagsToRemove;
        this.tagsToAdd = tagsToAdd;
    }

    public String getImagemUrl() {
        return imagemUrl;
    }

    public void setImagemUrl(String imagemUrl) {
        this.imagemUrl = imagemUrl;
    }

    public String[] getTagsToRemove() {
        return tagsToRemove;
    }

    public void setTagsToRemove(String[] tagsToRemove) {
        this.tagsToRemove = tagsToRemove;
    }

    public String[] getTagsToAdd() {
        return tagsToAdd;
    }

    public void setTagsToAdd(String[] tagsToAdd) {
        this.tagsToAdd = tagsToAdd;
    }

    
    
}
